packages/admin/cms-admin/src/generator/future/generator.ts

```diff
@@ -56,6 +56,10 @@ export type GridConfig<T extends { __typename?: string }> = {
     delete?: boolean;
     copyPaste?: boolean;
     readOnly?: boolean;
+    actions?: {
```

Comment:
for actions (plural) I would expect I can specify an array for multiple actions

Reply:
You can specify a component, that has multiple actions 😄 

Should we change this to two settings?

- `actionsComponent?: ImportReference`
- `actionsWidth?: number`

Reply:
@nsams should I do it the way I suggested?